Clarify docker download and start instructions in README

After downloading docker and docker-compose, I followed along the README, until `$ make up` produced the following: 

``` sh
chipy.org MacBook-Pro-3:chipy.org patrickboland4$ make up
docker-compose up -d
make: docker-compose: Permission denied
make: *** [up] Error 1
```

I received the same error after running `$ sudo make up`. 

The solution was, I needed to run the docker application on my Mac, which prompted me for the root password. 

I'd like to provide just a bit more clarity around starting Docker, especially for folks who are less familiar with Docker (like me).

Yes, we should definitely do that.  I don't have a mac to test on so help
there would be appreciated. I can add additional information for linux and
windows prior to WSL.


On Sat, Mar 6, 2021 at 12:49 PM Patrick Boland <notifications@github.com>
wrote:

> After downloading docker and docker-compose, I followed along the README,
> until $ make up produced the following:
>
> chipy.org MacBook-Pro-3:chipy.org patrickboland4$ make up
> docker-compose up -d
> make: docker-compose: Permission denied
> make: *** [up] Error 1
>
> I received the same error after running $ sudo make up.
>
> The solution was, I needed to run the docker application on my Mac, which
> prompted me for the root password.
>
> I'd like to provide just a bit more clarity around starting Docker,
> especially for folks who are less familiar with Docker (like me).
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/chicagopython/chipy.org/issues/383>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AA2KPYBUPRJNWN3UQ77VW43TCJ2KPANCNFSM4YW75WXQ>
> .
>

--- a/README.md
+++ b/README.md
@@ -46,6 +46,8 @@ Make the project directory your working directory:
 
     cd chipy.org
 
+**Note: The `make` commands require Make to be installed, and Docker must be running. Please follow the [Installation](#installation) guide before running the below steps**
+
 Run the setup command to configure the environment. This will copy
 a default configuration file from docker/docker.env.sample to
 docker/docker.env.

